### DISCUSSION TITLE
Not work very well in vscode.

### DISCUSSION THREAD

**Discussion by sbwcwso** [marlonrichert/zsh-autocomplete#240] [General]
In the vscode integrated terminal, when I type something it shows `zsh: do you wish to see all 115 possibilities (13 lines)?` and Interrupted my input。When I disable the plugin, this notice won't show.

And at the system terminal, everygthings works fine.

**Comment by marlonrichert** [comment]
Should be fixed now with 13cb2b515c3e9152fdd86d3fa8d70f23bf657563. Please try again.
